src/Templates/src/templates/maui-mobile/Pages/ProjectDetailPage.xaml.cs

```diff
@@ -0,0 +1,24 @@
+using MauiApp._1.Models;
+
+namespace MauiApp._1.Pages;
+
+public partial class ProjectDetailPage : ContentPage
+{
+	public ProjectDetailPage(ProjectDetailPageModel model)
+	{
+		InitializeComponent();
+
+		BindingContext = model;
+	}
+}
+
+public class ChipDataTemplateSelector : DataTemplateSelector
```

Comment:
can this class have its own file?

Reply:
Going to leave this one here for now we can discuss for the next iteration